### PR TITLE
Misc. updates to common, util-prov

### PR DESCRIPTION
**Pull request by jithinjosepkl**
- Minor updates to util-cq (default CQ size, fix bug in tagged-cq-read)
- Add new control packet types in fi_proto (NACK, DISCARD)
- Expose util-AV interface to report AV events
- Add util-function to get time in us
- Add util-function to get node name

### CODE REVIEW

**Comment by lanl-ompi** [comment]
Test FAILed.

**Comment by jithinjosepkl** [comment]
bot:retest

File: include/fi_proto.h [hunk 1]
```diff
@@ -56,6 +56,8 @@ enum {
 	ofi_ctrl_data,
 	ofi_ctrl_large_data,
 	ofi_ctrl_ack,
+	ofi_ctrl_nack,
+	ofi_ctrl_discard,
```
Comment (j-xiong) [inline include/fi_proto.h]: Would the new types cause the OFI_CTRL_VERSION to change?

Comment (jithinjosepkl) [inline include/fi_proto.h]: Hmm, I think we can keep the version as it is, since none of the providers use this currently.

**Comment by j-xiong** [comment]
👍

**Pull request merged**